OpenIV/Resources/Languages/zh_TW/LIP.xml

```diff
@@ -1,9 +1,6 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<LanguageInterfacePack version='290' ID='1028' active="true">
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<LanguageInterfacePack version='300' ID='2057' active="true">
 	<Contributors>
-		<Contributor link="http://openiv.com/">ky0ncheng</Contributor>
+		<Contributor link="http://openiv.com/">OpenIV 團隊</Contributor>
```

Comment:
You put "OpenIV Team" here. But, we didn't contribute for this language. Please change it.